### DISCUSSION TITLE
2022.08.20 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#35] [Self-Assessment]
### 이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.

ex) 
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점 
이유 : (생략)

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 자바 [클래스와 객체, 접근제한자, 상속] 단원 강의 완강 후 노션 정리

점수 : 0점. 
이유 : 이번 주는 자바공부를 못했어요... 오늘 카페가서 자바 강의 수강하고 정리해서 9시 전까지 인증 올리겠습니다.
2022.08.20 20:37 - 클래스, 객체, 접근제한자 정리 완료

**Comment by YoungSeok222222** [comment]
이번주 목표: 민코딩 진도 따라가기(문제 풀기)

점수: 60점
이유:  민코딩 진도를 따라잡기는  했지만, 이번주꺼까지 완벽하게 풀지는 못했기 때문에 일요일까지는 최대한 풀겠습니다.

**Comment by swjiae** [comment]
이번 주 목표 : 월ˑ수 노션 정리해서 커밋하기, 과제 당일에 끝내기, 홈트 2회

점수 : 50점
이유 : 노션정리 (2/2)  커밋 (1/2)
          과제 (2/3) - 주말동안 끝낼 예정
          홈트 (2/2) - 1회는 집걸어가기로 대체..

**Comment by sooyeonlee127** [comment]
이번 주 목표 : 수업 내용 티스토리 정리 및 커밋 2회 / 운동 2회
점수 : 60점
이유 : 
커밋 (2/2)
티스토리 정리 (0/2) - 블로그 만들기만 함 - 커밋한 내용 더 정리해서 주말 동안 올리겠습니다
홈트 (2/2)

**Comment by HyeseungNA** [comment]
이번 주 목표 : 수업 복습, 민코딩 1일 최소 1 레벨 풀기, 부족한 개념 노션에 정리
점수 : 70점
이유: 수업 복습과 수업 내용 노션에 올리는 것은 잘 했으나 민코딩 진도가 느리고 부족한 개념을 구글링하는 것에서 멈춰서 70점으로 줬습니다.

**Comment by moonthree** [comment]
이번 주 목표 : 기술 블로그 만들고 글 1개

점수 : 50점
이유 : 블로그는 만들고 글도 올리긴 했지만 노션에 정리한 걸 블로그에 옮기는 게 깔끔하게 안되서
수정 할 게 많고 블로그도 꾸준히 꾸며야 할 것 같습니다.
https://moon-coding.tistory.com/

**Reply by itmakesmesoft** [reply]
ㅋㅋㅋㅋㅋㅋㅋ문코딩 너무 센스있네
